docker-compose.yml

```diff
@@ -12,4 +14,19 @@ services:
       - ./tasks:/home/app/tasks
       - ./webspicy:/home/app/webspicy
     ports:
-      - "3000:3000"
+      - "4567:4567"
+  printit-weasyprint:
+    build:
+      context: .
+      dockerfile: Dockerfile-weasyprint
+    volumes:
+      - ./Rakefile:/home/app/Rakefile
+      - ./config.ru:/home/app/config.ru
+      - ./config:/home/app/config
+      - ./lib:/home/app/lib
+      - ./public:/home/app/public
+      - ./spec:/home/app/spec
+      - ./tasks:/home/app/tasks
+      - ./webspicy:/home/app/webspicy
+    ports:
+      - "4567:4567"
```

Comment:
Move everything to port 80 (at least in the images, if not on the port mapping).
How about `printit.princexml` and `printit.weasyprint` ?

Reply:
Done.